tools/pre-commit

```diff
@@ -8,6 +8,9 @@ LINTED_DIRECTORIES="apps shared build/test/unit tv_apps"
 JSHINT_EXECUTABLE="node_modules/.bin/jshint"
 JSHINT_XFAIL_LIST="build/jshint/xfail.list"
 JSHINT_ARGS="--reporter=build/jshint/xfail $JSHINT_ARGS"
+ESLINT_EXECUTABLE="node_modules/.bin/eslint"
+ESLINT_ARGS="--ignore-path .jshintignore -f compact -c .eslintrc"
+ESLINT_XFAIL_LIST="build/eslint/xfail.list"
```

Comment:
we'll need to handle the xfail list when running with the makefile too... otherwise we'll see the issues in the CI.